subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,54 @@
+from commands2 import SubsystemBase
+from wpilib import PWMVictorSPX, PneumaticHub, PneumaticsModuleType, RobotBase, Solenoid
+import constants
+
+
+class IntakeSubsystem(SubsystemBase):
+    def __init__(self) -> None:
+        SubsystemBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intakeDeployed = False  # default to intake retracted and not reverseed
+        self.intakeReversed = False
```

Comment:
seems unused -  the subsystem should handle all subsystem variables and other commands just call to those functions

Reply:
Did this ever get fixed or no lol

Reply:
yes